gen-app-builder/data-store-status-checker/examples/data_store_checker.ipynb

```diff
@@ -0,0 +1,433 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "7be72547515d"
+      },
+      "source": [
+        "# Data Store Status Checker\n",
+        "\n",
+        "<table align=\"left\">\n",
+        "\n",
+        "  <td>\n",
+        "    <a href=\"https://colab.research.google.com/github/GoogleCloudPlatform/generative-ai/blob/main/retrieval-augmented/enterprise-search/examples/question_answering.ipynb\">\n",
```

Comment:
Can change these badge links to point to the (soon to be) published location.

Reply:
Ack